src/components/Transaction/Transaction.spec.js

```diff
@@ -0,0 +1,71 @@
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import Transaction from './';
+
+const change = 42.42;
+const date = '2022-07-07';
+const note = 'cloth';
+const category = 'hobbies and free time';
+const editTxt = 'Edit';
+const deleteTxt = 'Delete';
+
+describe('Transaction', () => {
+	it('should render correctly', () => {
+		render(
+			<Transaction change={change} date={date} note={note} category={category}>
+				<button>{editTxt}</button>
+				<button>{deleteTxt}</button>
+			</Transaction>
```

Comment:
This should not make a difference. Your `Transaction` component doesn't accept `children`.
```suggestion
			<Transaction change={change} date={date} note={note} category={category} />
```